google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/project/ProjectSelector.java

```diff
@@ -234,9 +234,10 @@ private void updateCloudProjectSelection(CloudProject selection) {
     Optional<CredentialedUser> loggedInUser =
         loginService.getLoggedInUser(selection.googleUsername());
 
-    if (!loginService.isLoggedIn()) {
+    if (!loggedInUser.isPresent()) {
```

Comment:
does this still cover the case where the user is not logged in (not just logged in to a different account)?

Reply:
Yes, this would be the same logic - logged in user will not be present.

Reply:
right, but in that case, there won't be an email to show in the message right?

Reply:
In case we have a cloud project already saved (i.e. deployment) it does not matter if not users are logged in, we would know which account we need to sign in to get access to that specific project.

Reply:
aha. so these branch of logic is exclusively for the case where there is a stored cloud project (with a username stored too). sounds good.